modules/graderStake/s1_util.go

```diff
@@ -40,6 +42,13 @@ func ValidateS1(entryhash []byte, extids [][]byte, height int32, content []byte)
 	}
 	o := &spr.S1Content{V2Content: *o2}
 
+	if exids[0][0] == 6 {
```

Comment:
variable is not correct. it should be `extids ` instead of `exids`.  @aktary

Reply:
😵